bin/pycbc_make_skymap

```diff
@@ -16,7 +16,8 @@ import pycbc
 from pycbc.filter import sigmasq
 from pycbc.io import live, WaveformArray
 from pycbc.types import TimeSeries, FrequencySeries, \
-        MultiDetMultiColonOptionAction, MultiDetOptionAppendAction
+        MultiDetMultiColonOptionAction, MultiDetOptionAppendAction, \
```

Comment:
@titodalcanton (getting to grips with commenting and reviewing, sorry if i've pinged you twice) Should the `MultiDet` options be placed on 3 lines or is this okay?

Reply:
I think this is fine. When in doubt about code style, [PEP8](https://www.python.org/dev/peps/pep-0008/) is the reference we try to follow.